{
 "bad": [
  "flask._request_ctx_stack.top",
  "flask.current_app.ldap3_login_manager",
  "flask_ldap3_login.AuthenticationResponseStatus.success",
  "flask_ldap3_login.__init__.AuthenticationResponse",
  "flask_ldap3_login.__init__.AuthenticationResponseStatus",
  "flask_ldap3_login.__init__.log",
  "flask_wtf.FlaskForm",
  "flask_wtf.FlaskForm.validate",
  "ldap3",
  "ldap3.ALL_ATTRIBUTES",
  "ldap3.ANONYMOUS",
  "ldap3.Connection",
  "ldap3.SYNC",
  "ldap3.Server",
  "ldap3.core.exceptions.LDAPInvalidCredentialsResult",
  "ldap3.utils.conv.escape_filter_chars",
  "logging",
  "logging.debug",
  "logging.getLogger",
  "wtforms",
  "wtforms.BooleanField",
  "wtforms.PasswordField",
  "wtforms.StringField",
  "wtforms.SubmitField",
  "wtforms.validators",
  "wtforms.validators.Required",
  "wtforms.validators.ValidationError"
 ],
 "deps": {
  "enum": [
   "privy/conda-forge/noarch/privy-6.0.0-py_0",
   "python/conda-forge/linux-64/python-3.5.4-0",
   "python/conda-forge/linux-64/python-3.5.4-3",
   "python/conda-forge/linux-64/python-3.6.10-he5300dc_1011_cpython",
   "python/conda-forge/linux-64/python-3.6.11-h425cb1d_0_cpython",
   "python/conda-forge/linux-64/python-3.6.7-h357f687_1006",
   "python/conda-forge/linux-64/python-3.8.5-h1103e12_6_cpython",
   "python/conda-forge/linux-64/python-3.8.5-h1103e12_8_cpython",
   "python/conda-forge/linux-64/python-3.9.0-h2a148a8_4_cpython"
  ],
  "flask": [
   "flask/conda-forge/linux-64/flask-0.11.1-py27_0",
   "flask/conda-forge/linux-64/flask-0.12-py27_0",
   "flask/conda-forge/linux-64/flask-0.12-py36_0",
   "flask/conda-forge/linux-64/flask-1.0.2-py27_0",
   "flask/conda-forge/linux-64/flask-1.0.2-py35_0",
   "flask/conda-forge/linux-64/flask-1.0.2-py36_0",
   "flask/conda-forge/noarch/flask-0.11.1-py_1",
   "flask/conda-forge/noarch/flask-1.0.2-py_1",
   "flask/conda-forge/noarch/flask-1.0.3-py_0",
   "flask/conda-forge/noarch/flask-1.1.0-py_0"
  ]
 }
}